microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Support multiple profiles under a single user data folder
+description: How to have different profiles of a WebView2 app running with different profiles under a single user data folder to achieve browsing data separation.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 10/11/2022
+---
+# Support multiple profiles under a single user data folder
+
+To achieve data separation, a WebView2 app can choose to use different user data folders for different WebView2 controls. However, in such way you'll have to be running multiple WebView2 runtime instances (each including a browser process and a bunch of child processes), which means much more consumption for system resources including memory, CPU footprint, disk space, so it is not desirable.
+
+Now we have a more desirable solution: multiple profile support, which enables a WebView2 app to have multiple profiles under a single user data folder, each profile has a dedicated profile folder to save browser data, which means separate browsing data storage, such as cookies, user preference settings, and cached resources. All the WebView2 controls associated with the same profile share the single profile folder. 
```

Comment:
Note: in a forked PR based on this PR, I'm moving this paragraph to the top, to lead with the current/latest recommended way instead of leading with a way that we are recommending not to use.

Reply:
Done, see 
https://github.com/MicrosoftDocs/edge-developer/pull/2245/files

Please close this comment thread.